harness/determined/common/experimental/determined.py

```diff
@@ -130,8 +133,13 @@ def create_experiment(
             config(string, pathlib.Path, dictionary): experiment config filename (.yaml)
                 or a dict.
             model_dir(string, optional): directory containing model definition. (default: ``None``)
-            includes (Iterable[Union[str, pathlib.Path]], optional): Additional files or
-            directories to include in the model definition.  (default: ``None``)
+            includes(Iterable[Union[str, pathlib.Path]], optional): Additional files or
+            directories to include in the model definition. (default: ``None``)
+            parent_id(int, optional): The id of the parent experiment. (default: ``None``)
```

Comment:
I don't actually know what a parent experiment is in the context of another experiment. Could you help me out with a link to docs, and then we can figure out what the right description of it would be here?

Reply:
I can't seem to find any related docs, but passing a parent experiment means we borrow the model definition of the parent experiment: https://github.com/determined-ai/determined/blob/4ae09875aba8976584fefbdd575408498afad68d/master/internal/core_experiment.go#L345

Reply:
Wow. You had to dig. Thanks. This is new information for me, and something I should probably know, too.

The answer you've discovered is confusing to me. Does this make it incompatible with passing a `model_dir`?

I'll put some energy into answering these things, too. Whatever we discover should be in the docstring.

Reply:
I believe when `parentId` is presented, we just ignore `model_dir`

Reply:
I think the new docs look great.